index.html

```diff
@@ -2209,218 +2283,223 @@ <h3><dfn>Writing or pushing content</dfn></h3>
             and return |p|.
           </li>
           <li>
-            If pushing data is not supported by the underlying <a>NFC Adapter</a>, then reject
-            |p| with a {{"NotSupportedError"}}
+            If pushing data is not supported by the underlying
+            <a>NFC Adapter</a>, then reject |p| with a {{"NotSupportedError"}}
             {{DOMException}} and return |p|.
           </li>
+          <li>
+            Let |signal:AbortSignal| be the |options|’ dictionary member
+            of the same name if present, or `null` otherwise.
+          </li>
           <li>
             If |signal|’s [= AbortSignal/aborted flag =] is set, then reject |p|
             with an {{"AbortError"}} {{DOMException}} and return |p|.
           </li>
           <li>
             If |signal| is not `null`, then
-            <a data-cite="dom#abortsignal-abort-algorithms">add the following abort steps</a> to |signal|:
+            <a data-cite="dom#abortsignal-abort-algorithms">
+            add the following abort steps</a> to |signal|:
               <ol>
                 <li>
-                  If the instance has already initiated NFC data transfer,
-                  and it can't be aborted, ignore the abort signal and return.
-                </li>
-                <li>
-                  Stop the instance's |timer| if it is active.
-                </li>
-                <li>
-                  Reject |p| with an {{"AbortError"}}
-                  {{DOMException}} and abort these steps.
+                  Run the <a>abort a pending push operation</a> on the
+                  <a>environment settings object</a>.
                 </li>
               </ol>
           </li>
           <li>
-            Run the following steps <a>in parallel</a>:
+            [=promise/React=] to |p|: if |p| was settled (fulfilled or
```

Comment:
I still like these using a <ul> for readability

Reply:
Using what for readability? (not visible)